api/api.go

```diff
@@ -176,8 +176,8 @@ func (a *API) ListenExecution(service string, f *ExecutionFilter) (*ExecutionLis
 }
 
 // SubmitResult submits results for executionID.
-func (a *API) SubmitResult(executionID string, outputKey string, outputs []byte) error {
-	exec, stateChanged, err := a.processExecution(executionID, outputKey, outputs)
+func (a *API) SubmitResult(executionID string, outputs []byte) error {
```

Comment:
Don't forget to add `error` next to `outputs` as parameter of `SubmitResult` so the execution can failed from an error from the service (and not only from an error from the Engine itself).

Reply:
This is breaking service's task that send back a result with "error" outputKey:
```
➜  core git:(feature/single-output-pt-6) ./dev-cli service dev https://github.com/mesg-foundation/service-ethereum-erc20/tarball/single-outputs
```

```
➜  core git:(feature/single-output-pt-6) ./dev-cli service execute ethereum-erc20 --task name --data contractAddress='0x420167d87d35c3a249b32ef6225872fbd9ab85d' 
✔ Task "name" executed
⢿ Waiting for result... ^C
```